src/viadot/orchestration/prefect/utils.py

```diff
@@ -57,10 +57,14 @@ def __init__(
             of the day X years ago
         - "X_years/months/days_ago_full_date":  e.g., "3_years_ago_full_date",
             refers to a given date X units ago in dynamic_date_format
+        - "years_from_x_until_now_included": e.g. "years_from_2019_until_now_included",
+           refers to a date range of the years from a given year
+           until the current year included
+        - "years_from_x_until_last_year": e.g., "years_from_2019_until_last_year",
```

Comment:
I would change the second option to `years_from_x_until_x_year_ago` (or something similar using a better name). It will be more useful in various cases.
example: `years_from_2019_until_2_years_ago`.

Please update it and add tests for new cases.